docs/learning/getting-started/rba/node-setup.md

```diff
@@ -0,0 +1,84 @@
+---
+
+prev:
+  link: '/learning/getting-started/rba/runner-setup.md'
+  text: "Create a Runner to connect to remote nodes"
+
+---
+
+# Add and connect to a node through the Runner
+
+Now that a Runner has been configured, the next goal is to run a job on a test machine in your environment.  This will require a bit more configuration.  These instructions are designed with a Linux machine in mind and you’ll need a few pieces of information to complete these steps:
+
+* An IP address or hostname that can be reached from the Runner
+* A private key (usually a .pem file) for connecting to the node with SSH 
+
+## Add key to keystore
+
+PagerDuty’s Process Automation products have a built-in keystore for storing keys and passwords.  In this section, you’ll upload the private key needed to connect from the Runner to the test node.
+
+1. Access Key Storage for the Project by clicking the Project Settings gear at the bottom left and then click Key Storage  
+![Choose key Storage under Project Settings](/assets/img/noderun1.png)  
+2. In Key Storage, click the **Add or Upload a Key** green button  
+![Click Add or Upload](/assets/img/noderun2.png)  
+3. Ensure that **Private Key** is selected for the **Key Type** then select **Upload File** from the menu at the left.  
+![Select Private Key](/assets/img/noderun3.png)  
+4. After selecting Upload File, click the **Choose File** button to browse and select a private .pem key for connecting to your test node through SSH  
+![Choose File](/assets/img/noderun4.png)  
+5. Enter a name for the key in the name field, such as **testkey**, then click the green Save button at the bottom  
+![Enter Name](/assets/img/noderun5.png)  
+
+## Add a test node
+
+Next, we’ll add your test machine to the project as a node.  Nodes can be defined manually or dynamically using Node Sources, which are configured under Project Settings.  In this case, as with most testing, we’ll add the node information using a source called Node Wizard.  if you don't see the Node Wizard, you can add it in by clicking **Edit Nodes | Sources | Add a new Node Source** under the **Project settings** gear at the bottom left.  
+
+6. Click the **Project Settings** gear at the bottom left, then **Edit Nodes**  
```

Comment:
The code says "6" but it will likely restart the numbering over at "1" since it's a new paragraph.

Reply:
It's displaying as I hoped in the local repo but I'll keep an eye on it,.

Reply:
maybe the new site version handles this better.